.docker/setup_config.sh

```diff
@@ -0,0 +1,24 @@
+echo "About to start conda update, this may take some time..."
+source setup/setup_conda.sh Linux-x86_64
+source setup/setup.sh
+conda clean -t
+conda clean -p
+
+if [ -d "webapp/www/" ]; then
+    cp /index.html webapp/www/index.html
+fi
+
+if [ -d "/conf" ]; then
+    echo "Found configuration, overriding..."
+    cp -r /conf/* conf/
+fi
+
```

Comment:
@aGuttman given that we only run `setup_config` while building the image, do we need still need this?
you are going to replace the `setup_config.sh` in the `webapp` and `analysis` containers with this copy, right?